homeassistant/components/asuswrt/router.py

```diff
@@ -334,7 +338,7 @@ async def update_devices(self) -> None:
         wrt_devices = {format_mac(mac): dev for mac, dev in api_devices.items()}
         for device_mac, device in self._devices.items():
             dev_info = wrt_devices.pop(device_mac, None)
-            device.update(dev_info, consider_home)
+            device.update(dev_info, consider_home)  # type: ignore[arg-type]
```

Comment:
Can you try to add typing to `dev_info` and and `consider_home`.
You might also need to check if `devce_info` is not None.
That would be better than adding `# type: ignore[arg-type]`.
Thanks

Reply:
Typing `consider_home` do not work, it always return back error wrong argument type (type object, expected int). I try this in different way, at the end I put the hint. Suggestion?
`dev_info` can be None, this is expected by `update` function.

Reply:
Ok, fixed this, removed hint `# type: ignore[arg-type]`.